### PR TITLE
[ROCm] guard ROCm-only files in NVFUSER_RUNTIME_FILES

### PR DESCRIPTION
**Pull request by jeffdaily**
Addresses comment in #82498 as a follow-up PR.  

https://github.com/pytorch/pytorch/pull/82498#discussion_r958745967

### CODE REVIEW

**Comment by jeffdaily** [comment]
@jjsjann123 let's wait for green CI signal before landing.

**Review by jjsjann123 — APPROVED** [review]
LGTM

**Pull request closed without merging**
